packages/website/src/components/lib/markdown.ts

```diff
@@ -49,10 +50,18 @@ export function createMarkdown(state: ConfigModel): string {
 }
 
 export function createMarkdownParams(state: ConfigModel): string {
+  const { rules = {} } = parseESLintRC(state.eslintrc);
```

Comment:
```suggestion
  const { rules } = parseESLintRC(state.eslintrc);
```

`rules` will be always defined as an object